modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.html

```diff
@@ -0,0 +1,30 @@
+<div class="align-items-center d-flex evp-process">
```

Comment:
Could you rename the class to: 
```suggestion
<div class="align-items-center d-flex evp-process-title">
```